EOL.rst

```diff
@@ -35,7 +35,7 @@ This mostly mimics Debian lifecycle.
 .. csv-table:: centos
    :header: "distro", "EOL", "LTS", "ELTS"
 
-   "centos stream9", "rolling", "", ""
+   "centos stream9", "~2027-05-31", "", ""
```

Comment:
```suggestion
   "centos stream9", "2027-05-31", "", ""
```

I guess it's probably subject to change but an isoformat string is required here to allow for date comparison later on.

Reply:
will be handled by automatic update.